src/main/kotlin/com/atlassian/performance/tools/report/api/FullReport.kt

```diff
@@ -79,11 +79,13 @@ class FullReport {
                 systemMetrics = result.systemMetrics
             )
 
-            val report = PlaintextReport(
-                ActionMetricStatistics(result.actionMetrics)
-            ).generate()
+            val actionStats = ActionMetricStatistics(result.actionMetrics)
+            val plaintextReport = PlaintextReport(actionStats)
+            val csvReport = CSVReport(actionStats)
```

Comment:
I think you missed `generate()` calls, didn't you?

```suggestion
            val plaintextReport = PlaintextReport(actionStats).generate()
            val csvReport = CSVReport(actionStats).generate()
```

Reply:
oops! yes, thanks!